monkey/monkey_island/cc/services/reporting/issue_processing/exploit_processing/exploiter_report_info.py

```diff
@@ -1,14 +1,7 @@
 from dataclasses import dataclass
-from enum import Enum
 from typing import Union
 
 
-class CredentialType(Enum):
```

Comment:
Also unused in `monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js:49`